particles/Chooser/Chooser.js

```diff
@@ -26,32 +26,52 @@
         this.emit('values', result);
       });
       this.when([new StateChanges('values'), new SlotChanges()], async e => {
-        let values = this.states.get('values');
-        if (values.length <= 0) {
-          this.releaseSlot('action');
+        let slotName = 'action';
```

Comment:
Yes, the slotName shouldn't be hardcoded but that's a different task, as you say.